src/components/Navigation/DropdownNav.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { within, userEvent } from "@storybook/test";
+import type { DropdownNavGroup } from "./DropdownNav";
+import { DropdownNav } from "./DropdownNav";
+
+const meta = {
+  title: "Components/DropdownNav",
+  component: DropdownNav,
+  parameters: {
+    viewport: {
+      defaultViewport: "mobile2",
```

Comment:
@winkerVSbecks I was wondering if you're ok with leaving out the `parameter` for now as it's introducing some inconsistent behavior with the other stories as it propagates it when navigating to other stories, which prevents the content from rendering correctly. When 8.3 lands and StoryGlobals are available, we can enable it for a specific set of stories, including this one, to test how it performs. Wouldn't you agree?

Reply:
yea, that's cause the nav is only visible on mobile. I'm going to have to add viewport back in.

Reply:
Actually found a solution for it. Just needed to set a global default.